azure/packages/test/end-to-end-tests/src/test/utils.ts

```diff
@@ -2,15 +2,19 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+import { IMember } from "fluid-framework";
+
 import { AzureMember, IAzureAudience } from "@fluidframework/azure-client";
 import { ISharedMap, IValueChanged } from "@fluidframework/map";
 
-export const waitForMyself = async (audience: IAzureAudience): Promise<AzureMember> => {
+export const waitForMyself = async (
```

Comment:
Since this is now generalized to wait for the specified userId, maybe rename to `waitForMember`